app/scripts/lib/app-start.js

```diff
@@ -110,6 +111,7 @@ function (
   User,
   FormPrefill,
   Notifications,
+  PageRefreshWatch,
```

Comment:
We have screens, we have views and we have pages. What's the difference between them, if any?